firebase-firestore/src/testUtil/java/com/google/firebase/firestore/testutil/TestUtil.java

```diff
@@ -395,8 +396,16 @@ public static RemoteEvent noChangeEvent(int targetId, int version, ByteString re
 
   public static RemoteEvent addedRemoteEvent(
       MaybeDocument doc, List<Integer> updatedInTargets, List<Integer> removedFromTargets) {
-    DocumentChange change =
-        new DocumentChange(updatedInTargets, removedFromTargets, doc.getKey(), doc);
+    return addedRemoteEvent(Collections.singletonList(doc), updatedInTargets, removedFromTargets);
```

Comment:
Note: the changes here are also part of https://github.com/firebase/firebase-android-sdk/pull/697/files#diff-ac4f307b0804f62b9719e8c7556226c2L396